tests/test_flow/test_logging.py

```diff
@@ -112,9 +119,48 @@ def log_unpickleable_value():
 
     assert builder.build().get("log_unpickleable_value") == 5
 
-    log_checker.expect(
+    log_checker.expect_all(
         "Computing  log_unpickleable_value() ...",
         "Cannot pickle me",
         "Logging unpickleable class: Cannot pickle me",
         "Computed   log_unpickleable_value()",
     )
+
+
+@pytest.mark.no_parallel
+@pytest.mark.needs_aip
+def test_log_aip(aip_builder, log_checker):
+    builder = aip_builder
+
+    builder.assign("x", 1)
+
+    @builder
+    @bn.aip_task_config("n1-standard-4")
+    def x_plus_one(x):
+        return x + 1
+
+    @builder
+    @bn.aip_task_config("n1-standard-8")
+    def x_plus_two(x_plus_one):
+        return x_plus_one + 1
+
+    flow = builder.build()
+
+    assert flow.get("x_plus_one") == 2
+
+    log_checker.contains_substrings(
+        "Accessed   x(x=1) from definition",
+        "Uploading x(x=1) to GCS ...",
+        "Started task on AI Platform: https://console.cloud.google.com/ai-platform/jobs/",
+        "Computed   x_plus_one(x=1)",
```

Comment:
Maybe this should say `Computed    x_plus_one(x=1) on AIP` or something to make it clear that it happened remotely?

Reply:
I am somewhat neutral to this. It seems redundant; we already have a bunch of logs for AIP when starting a job.

Reply:
We do, but the `Computed` message won't necessarily appear right after the `Started task` message; there could be several other messages in between, so it seems like it won't necessarily be obvious where the computation happened.

Reply:
Updated to show `Computed ... using AIP`